File: JULIA/Cap04/c04ex07.jl
# c04ex07.jl

print("Entre um numero inteiro: ")
numero = parse(Int, readline())

if (numero >= 20) && (numero <= 90)
  println("O numero esta na faixa de 20 a 90.")
else
  println("O numero esta fora da faixa de 20 a 90.")
end

println()
print("Tecle <Enter> para encerrar... ")
readline()

File: CSHARP/Cap04/c04ex07.cs
// c04ex07.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        int Numero;

        Write("Entre um numero inteiro: ");
        Numero = int.Parse(ReadLine());

        if ((Numero >= 20) && (Numero <= 90))
            WriteLine("O numero esta na faixa de 20 a 90.");
        else
            WriteLine("O numero esta fora da faixa de 20 a 90.");

        Write("\nAperte <Enter> para encerrar... ");
        ReadLine();
    }
}
